src/css/popup.css

```diff
@@ -40,20 +40,23 @@
   margin-bottom: 0;
 }
 
+.fx-relay-panel-content.phones-stats, 
+.fx-relay-panel-content.emails-stats {
+  padding: var(--spacingSm) var(--spacingMd);
+}
+
 /* FIXME: Refactor to account for height dynamically */
 /* Magic Number: Custom max-height for masks panel */
 #masks-panel[data-account-level="free"] .fx-relay-panel-content {
   min-height: 344px;
   max-height: 344px;
-  margin-bottom: 50px;
   overflow: auto;
 }
 
 #masks-panel[data-account-level="premium"] .fx-relay-panel-content {
-  min-height: 336px;
-  max-height: 336px;
+  min-height: 315px;
+  max-height: 315px;
```

Comment:
question (non-blocking): Why did this change? Just curious.

Reply:
Removes the second scroll bar that appears if the height is bigger in the emails panel